modules/aioseop_sitemap.php

```diff
@@ -2727,6 +2966,19 @@ private function get_archive_prio_from_posts( $posts ) {
 			}
 
 			$archives	= array();
+
+			/**
+			 * Filter to include additional post type archives in the sitemap.
+			 *
+			 * @since ?
+			 *
+			 * @param array $types {
+			 *      Key/value array, where key corresponds to the post type and value to the post object.
+			 *
+			 *      @type   string  The post type.
```

Comment:
Just the WP_Post variable needed to be documented in the array.

```
/**
 * Filter to include additional post type archives in the sitemap.
 *
 * @since 2.9
 *
 * @type array $types {
 *     Key/value array, where key corresponds to the post type and value to the post object.
 *
 *     @type WP_Post $post_type {
 *         The post object.
 *
 *         @see WP_Post
 *         @link https://codex.wordpress.org/Class_Reference/WP_Post
 *     }
 * }
 */
```